src/lib/y2users/user_validator.rb

```diff
@@ -90,5 +91,127 @@ def add_fatal_issue(list, error, location)
 
       list << Y2Issues::Issue.new(error, location: location, severity: :fatal)
     end
+
+    MIN_LENGTH = 2
+    # reason: see for example man utmp, UT_NAMESIZE
+    MAX_LENGTH = 32
+    def check_length
+      return _("No username entered.\nTry again.") if user.name.nil? || user.name.empty?
+
+      return "" if (MIN_LENGTH..MAX_LENGTH).include?(user.name.size)
+
+      format(_("The username must be between %i and %i characters in length.\n" \
+        "Try again."), MIN_LENGTH, MAX_LENGTH)
+    end
+
+    LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz".freeze
+    NUMBERS = "0123456789".freeze
+    # Regexp for allowed characters.
+    # NOTE: this is based on default in login.defs, maybe read it on running system?
+    CHAR_REGEXP = /\A[#{LETTERS}_][#{LETTERS}#{NUMBERS}_.-]*[#{LETTERS}#{NUMBERS}_.$-]?\z/.freeze
+    def check_characters
+      return "" if user.name =~ CHAR_REGEXP
+
+      _("The username may contain only\n" \
+        "letters, digits, \"-\", \".\", and \"_\"\n" \
```

Comment:
In Russian, Я is a letter.

```suggestion
        "Latin letters and digits, \"-\", \".\", and \"_\"\n" \
```

Reply:
good note...so we need to change it in both UsersSimple and here.